formal/promela/models/threadq/docs/notes/Intro.tex

```diff
@@ -0,0 +1,7 @@
+\section{Introduction}
```

Comment:
The integration of this copy and paste stuff from various sources is a bit problematic. Also the files in the "doc" directory have not SPDX Licence Identifier. I think the integration of the documentation should be done at a later stage and maybe to the rtems-docs repository.

Reply:
I agree - most of this is for my own use, anyway.